mava/components/building/best_checkpointer.py

```diff
@@ -21,6 +21,11 @@ class BestCheckpointerConfig:
 
 
 class BestCheckpointer(Component):
+    """Best checkpointer is a component that help in storing the best network
+    params for two options:
```

Comment:
Minor typo: 
`Best checkpointer is a component that help in storing the best network params for two options` -> `A component that stores the best network paramaters for two options`

Reply:
Done!